go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -1474,6 +1474,24 @@ func TestDescribeTableAsOf(t *testing.T) {
 	enginetest.TestScript(t, h, DescribeTableAsOfScriptTest)
 }
 
+func TestDescribeTableAsOfPrepared(t *testing.T) {
+	h := newDoltHarness(t)
+	defer h.Close()
+	enginetest.TestScriptPrepared(t, h, DescribeTableAsOfScriptTest)
+}
+
+func TestDescribeTableAsOfWithFk(t *testing.T) {
+	h := newDoltHarness(t)
+	defer h.Close()
+	enginetest.TestScript(t, h, DescribeTableAsOfWithFkScriptTest)
+}
+
+func TestDescribeTableAsOfWithFkPrepared(t *testing.T) {
+	h := newDoltHarness(t)
+	defer h.Close()
+	enginetest.TestScriptPrepared(t, h, DescribeTableAsOfWithFkScriptTest)
+}
```

Comment:
(minor) It would be nice to add this one new script test to the existing `TestDescribeTableAsOf` test, just to keep the top level Test functions a little more consolidated. (If we had just one ScriptTest in each top level Test function, it would be a bit unwieldily)

Reply:
Tests consolidated.